en/docs/extended_topics/consistent_mirrored.md

```diff
@@ -1,10 +1,10 @@
 ## The view of Consistent and Mirrored
 
-When doing distributed training, OneFlow provides two aspects for determining the relationship between data and models. There are `consistent` strategy and `mirrored` strategy.
+In distributed training, OneFlow provides two aspects for determining the relationship between data and model. There are `consistent` strategy and `mirrored` strategy.
```

Comment:
There are `consistent` strategy and `mirrored` strategy.
->
There are `consistent` view and `mirrored` view.